packages/replay/test/unit/coreHandlers/util/truncateJson/fixJson.test.ts

```diff
@@ -0,0 +1,65 @@
+import { fixJson } from '../../../../../src/util/truncateJson/fixJson';
+
+describe('Unit  | coreHandlers | util | truncateJson | fixJson', () => {
+  test.each([
+    // Basic steps of object completion
+    ['{', '{"~~":"~~"}'],
+    ['{}', '{}'],
+    ['{"', '{"~~":"~~"}'],
+    ['{"a', '{"a~~":"~~"}'],
+    ['{"aa', '{"aa~~":"~~"}'],
+    ['{"aa"', '{"aa":"~~"}'],
+    ['{"aa":', '{"aa":"~~"}'],
+    ['{"aa":"', '{"aa":"~~"}'],
+    ['{"aa":"b', '{"aa":"b~~"}'],
+    ['{"aa":"bb', '{"aa":"bb~~"}'],
+    ['{"aa":"bb"', '{"aa":"bb","~~":"~~"}'],
+    ['{"aa":"bb"}', '{"aa":"bb"}'],
+
+    // Basic steps of array completion
+    ['[', '["~~"]'],
+    ['[]', '[]'],
+    ['["', '["~~"]'],
+    ['["a', '["a~~"]'],
+    ['["aa', '["aa~~"]'],
+    ['["aa"', '["aa","~~"]'],
+    ['["aa",', '["aa","~~"]'],
+    ['["aa","', '["aa","~~"]'],
+    ['["aa","b', '["aa","b~~"]'],
+    ['["aa","bb', '["aa","bb~~"]'],
+    ['["aa","bb"', '["aa","bb","~~"]'],
+    ['["aa","bb"]', '["aa","bb"]'],
+
+    // Nested object/arrays
+    ['{"a":{"bb', '{"a":{"bb~~":"~~"}}'],
+    ['{"a":["bb",["cc","d', '{"a":["bb",["cc","d~~"]]}'],
```

Comment:
Maybe a few test cases with an array of nested objects

Reply:
true, can add a few more here!

Reply:
I added some more tests, esp. with a large JSON body of nested stuff (found some issues while doing this, great!)